### PR TITLE
Update View Transition API data (update Safari support, add Firefox impl_url, remove outdated impl_url)

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Summary

note that Safari has implement View Transition Level 1 and is working on View Transition Level 2

see also https://github.com/mdn/browser-compat-data/pull/24870

### CODE REVIEW

**Comment by caugner** [comment]
@skyclouds2001 Can you please take a look at the conflicts? 🙏

File: api/Window.json [hunk 1]
```diff
@@ -3856,7 +3855,8 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": false,
+              "impl_url": "https://webkit.org/b/263602"
```
Comment (skyclouds2001) [inline api/Window.json:3859]:
```suggestion
              "version_added": "18.2"
```

File: api/Window.json [hunk 2]
```diff
@@ -3934,7 +3933,8 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": false,
+              "impl_url": "https://webkit.org/b/269904"
```
Comment (skyclouds2001) [inline api/Window.json:3937]:
```suggestion
              "version_added": "18.2"
```

**Comment by skyclouds2001** [comment]
the https://github.com/mdn/browser-compat-data/pull/24939 seems to miss api.Window.pageswap_event & api.Window.pagerevert_event

as test the data locally:

image

https://bugs.webkit.org/show_bug.cgi?id=277524
https://bugs.webkit.org/show_bug.cgi?id=277940
https://github.com/WebKit/WebKit/pull/32025
https://github.com/WebKit/WebKit/pull/31622

**Review by caugner — APPROVED** [review]

**Pull request merged**